aspnet-core/xml/FrameworksIndex/aspnetcore-7.0.xml

```diff
@@ -4,8 +4,8 @@
     <Assembly Name="Microsoft.AspNetCore.Antiforgery" Version="7.0.0.0" />
     <Assembly Name="Microsoft.AspNetCore.ApiAuthorization.IdentityServer" Version="7.0.0.0" />
     <Assembly Name="Microsoft.AspNetCore.Authentication.Abstractions" Version="7.0.0.0" />
-    <Assembly Name="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="7.0.0.0" />
-    <Assembly Name="Microsoft.AspNetCore.Authentication.AzureADB2C.UI" Version="7.0.0.0" />
+    <Assembly Name="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="6.0.11.0" />
+    <Assembly Name="Microsoft.AspNetCore.Authentication.AzureADB2C.UI" Version="6.0.11.0" />
```

Comment:
This continues to look wrong. We no longer ship these assemblies. In the previous PR, I asked @HaoK to confirm but the files are definitely not part of 7.0.x.

Not just deleting these lines also explains many of the `6.0.11` additions in files just below.

Reply:
This is happening because the Microsoft.AspNetCore.Authentication.AzureAD.UI and Microsoft.AspNetCore.Authentication.AzureADB2C.UI packages were still in the configuration for 7 which is set up to pull in the latest version. 
Are there other packages that should be removed?

Reply:
Full list of packages removed since 6.0.x:
- AspNetCoreRuntime.7.0.x64 (it's AspNetCoreRuntime.7.0.x64 in this release)
- AspNetCoreRuntime.6.0.x86 (it's AspNetCoreRuntime.7.0.x86 in this release)
- Microsoft.AspNetCore.Authentication.AzureAD.UI
- Microsoft.AspNetCore.Authentication.AzureADB2C.UI
- Microsoft.AspNetCore.Mvc.Razor.Extensions
- Microsoft.AspNetCore.Razor.Language
- Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv
- Microsoft.CodeAnalysis.Razor
- Microsoft.DotNet.Web.ItemTemplates.6.0 (it's Microsoft.DotNet.Web.ItemTemplates.7.0 in this release)
- Microsoft.DotNet.Web.ProjectTemplates.6.0 (it's "Microsoft.DotNet.Web.ProjectTemplates.7.0 in this release)
- Microsoft.DotNet.Web.Spa.ProjectTemplates.6.0 (it's Microsoft.DotNet.Web.Spa.ProjectTemplates.7.0 in this release)

New packages since 6.0.0:
- Microsoft.AspNetCore.Components.CustomElements
- Microsoft.AspNetCore.Grpc.JsonTranscoding
- Microsoft.AspNetCore.Grpc.Swagger
- Microsoft.AspNetCore.OpenApi

Let us know if you need contacts to review any of the APIs from the new packages (assembly names should match).

Reply:
@dougbu Thank you! 

It looks like we were only missing Microsoft.AspNetCore.Grpc.Swagger. 

From previous runs I believe AspNetCoreRuntime.7.0.x64, and AspNetCoreRuntime.7.0.x86 duplicate Microsoft.AspNetCore.App.Ref which is what we use for the bulk of the ASP libraries.

The CI job has been run again with the following updated list of packages:

- Microsoft.AspNetCore.App.Ref
- Microsoft.AspNetCore.ApiAuthorization.IdentityServer
- Microsoft.AspNetCore.Authentication.Certificate
- Microsoft.AspNetCore.Authentication.Facebook
- Microsoft.AspNetCore.Authentication.Google
- Microsoft.AspNetCore.Authentication.JwtBearer
- Microsoft.AspNetCore.Authentication.MicrosoftAccount
- Microsoft.AspNetCore.Authentication.Negotiate
- Microsoft.AspNetCore.Authentication.OpenIdConnect
- Microsoft.AspNetCore.Authentication.Twitter
- Microsoft.AspNetCore.Authentication.WsFederation
- Microsoft.AspNetCore.Authorization
- Microsoft.AspNetCore.AzureAppServices.HostingStartup
- Microsoft.AspNetCore.AzureAppServicesIntegration
- Microsoft.AspNetCore.Components
- Microsoft.AspNetCore.Components.Authorization
- Microsoft.AspNetCore.Components.CustomElements
- Microsoft.AspNetCore.Components.Forms
- Microsoft.AspNetCore.Components.Web
- Microsoft.AspNetCore.Components.WebAssembly
- Microsoft.AspNetCore.Components.WebAssembly.Authentication
- Microsoft.AspNetCore.Components.WebAssembly.DevServer
- Microsoft.AspNetCore.Components.WebAssembly.Server
- Microsoft.AspNetCore.Components.WebView
- Microsoft.AspNetCore.ConcurrencyLimiter
- Microsoft.AspNetCore.Connections.Abstractions
- Microsoft.AspNetCore.Cryptography.Internal
- Microsoft.AspNetCore.Cryptography.KeyDerivation
- Microsoft.AspNetCore.DataProtection
- Microsoft.AspNetCore.DataProtection.Abstractions
- Microsoft.AspNetCore.DataProtection.EntityFrameworkCore
- Microsoft.AspNetCore.DataProtection.Extensions
- Microsoft.AspNetCore.DataProtection.StackExchangeRedis
- Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore
- Microsoft.AspNetCore.Grpc.JsonTranscoding
- Microsoft.AspNetCore.Grpc.Swagger
- Microsoft.AspNetCore.HeaderPropagation
- Microsoft.AspNetCore.Hosting.WindowsServices
- Microsoft.AspNetCore.Http.Connections.Client
- Microsoft.AspNetCore.Http.Connections.Common
- Microsoft.AspNetCore.Identity.EntityFrameworkCore
- Microsoft.AspNetCore.Identity.UI
- Microsoft.AspNetCore.JsonPatch
- Microsoft.AspNetCore.Metadata
- Microsoft.AspNetCore.MiddlewareAnalysis
- Microsoft.AspNetCore.Mvc.NewtonsoftJson
- Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation
- Microsoft.AspNetCore.Mvc.Testing
- Microsoft.AspNetCore.OpenApi
- Microsoft.AspNetCore.Owin
- Microsoft.AspNetCore.RateLimiting
- Microsoft.AspNetCore.SignalR.Client
- Microsoft.AspNetCore.SignalR.Client.Core
- Microsoft.AspNetCore.SignalR.Common
- Microsoft.AspNetCore.SignalR.Protocols.Json
- Microsoft.AspNetCore.SignalR.Protocols.MessagePack
- Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson
- Microsoft.AspNetCore.SignalR.Specification.Tests
- Microsoft.AspNetCore.SignalR.StackExchangeRedis
- Microsoft.AspNetCore.SpaProxy
- Microsoft.AspNetCore.SpaServices.Extensions
- Microsoft.AspNetCore.TestHost
- Microsoft.Authentication.WebAssembly.Msal
- Microsoft.Extensions.Identity.Core
- Microsoft.Extensions.Identity.Stores
- Microsoft.JSInterop
- Microsoft.JSInterop.WebAssembly
- Microsoft.Extensions.Features